src/more-infos/more-info-updater.html

```diff
@@ -0,0 +1,55 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="more-info-updater">
+  <style>
+    .data-entry .value {
+      max-width: 200px;
+    }
+    .center {
+      text-align: center;
+    }
+    .centered {
+      margin: auto;
+    }
+    .divider {
+      padding: 5px;
+    }
+    .link {
+      color: #03A9F4;
+    }
+  </style>
+  <template>
+    <div class='layout vertical'>
+
+      <!-- GitHub Link -->
+      <div class='data-entry layout justified horizontal center' style="width:100%" id='setting'>
```

Comment:
`width: 100%` is the default value for `div` because it is `display: block`. I also think that all the classes here are not needed if you just want to center it, the `center `class is enough. I also think the `id` is unused?